### PR TITLE
TSV downloads performance improvement [GAWB-2350]

### PR DESCRIPTION
**Pull request by aednichols**
Orchestration specifies a sort field in its queries to Rawls, while Alex's Python code does not. Rawls does an extra join when the sort is applied, blowing up execution time.

We can [avoid the join by sorting specifically on entity name](https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala#L138). It's already the default in at least one [place](https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiService.scala#L41).

I don't suppose the sort order matters in the final product, but the MD5 of the 20k workspace file is the same before and after this PR (`c38e074a85d88537b8327e60ee1d005b`).

The running time for 20k is down from ~10 minutes to ~2 minutes.

---
This PR represents the smallest possible change. We could also...
1. Make `sortField` and `sortDirection` into `Option` types on `EntityQuery`, with more far-reaching consequences.
2. Overload `entityQueryUriFromWorkspaceAndQuery` in Orchestration with a version that takes a `Map` instead of an `EntityQuery` to construct the URL.

---
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [x] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by rushtong** [comment]
The time savings is exactly what Alex was hoping for. That buys us a good deal of time until we start hitting the current 15 minute LB timeout settings.

**Review by rushtong — APPROVED** [review]
👍  LGTM, and also for keeping this small. I prefer that over the more complicated model changes.

**Comment by rushtong** [comment]
Any chance we can debug this failing travis test:
```
[info] HttpOntologyDAOConcurrencySpec:
[info] HttpOntologyDAO concurrency
[info] - should obey the spray max-clients throttle *** FAILED ***
[info]   Expected 8, but got 4 request count should have reached its maximum throttle limit (HttpOntologyDAOConcurrencySpec.scala:68)
```

**Comment by aednichols** [comment]
I inspected that test and couldn't come up with much; could not get it to fail again. It seems unlikely that out of 40 requests (`throttleCount * 5`) 36 would somehow get lost, so maybe the problem is with counting those received?

File: src/main/scala/org/broadinstitute/dsde/firecloud/service/ExportEntitiesByTypeActor.scala [hunk 1]
```diff
@@ -254,7 +254,7 @@ class ExportEntitiesByTypeActor(rawlsDAO: RawlsDAO,
   private def getEntityQueries(metadata: EntityTypeMetadata, entityType: String): Seq[EntityQuery] = {
     val pageSize = FireCloudConfig.Rawls.defaultPageSize
     val filteredCount = metadata.count
-    val sortField = entityType + "_id"
+    val sortField = "name" // Anything else and the Rawls execution time blows up with a join
```
Comment (ahaessly) [inline src/main/scala/org/broadinstitute/dsde/firecloud/service/ExportEntitiesByTypeActor.scala]: looks good. maybe reference the ticket number?

**Review by ahaessly — APPROVED** [review]

**Pull request merged**